### ISSUE TITLE
zig fmt misplaces closing brace when formatting functions with unused args

### ISSUE THREAD

**Issue by bingcicle** [ziglang/zig#14624]
### Zig Version

0.11.0-dev.1265+3ab43988c

### Steps to Reproduce and Observed Behavior

1. Copy paste: `mkdir fmt-bug && cd fmt-bug && zig init-exe`
2. Within `src/main.zig`, create a random function with an unused arg, for example,

```zig
pub fn fmt_bug(foo: u8) void {}
```

After formatting,

```zig
pub fn fmt_bug(foo: u8) void {
  _ = foo;}
```

Of course this is trivial to fix because saving again fixes the placement of the closing brace.

### Expected Behavior

The closing brace should be inserted on the next line,

```zig
pub fn fmt_bug(foo: u8) void {
  _ = foo;
}
```

**Comment by Vexu** [comment]
Zig doesn't fix unused arguments yet (see #12803), this is a [zls](https://github.com/zigtools/zls/) bug.

**Comment by bingcicle** [comment]
Ah sorry about opening this then! Thanks @Vexu
